src/Microsoft.Health.SqlServer/Routing/KnownRoutes.cs

```diff
@@ -15,10 +16,17 @@ public static class KnownRoutes
     public const string Versions = "versions";
 
     public const string Current = Versions + "/current";
-    public const string Script = Versions + "/{id:int}/script";
+    public const string IdSegment = "{id:int}";
```

Comment:
Do you know why this is a route template? Is this because the server component is using this same class for its route? I'd probably prefer to have separate members for the client and server instead of trying to replace it to create a format string for the client

Reply:
I'm guessing it's a route template because of its use in the AvailableVersion SQL Server API: https://github.com/microsoft/healthcare-shared-components/blob/eeb2beb077ba0798ae81e28745345aa23731fe72/src/Microsoft.Health.SqlServer.Api/Controllers/SchemaController.cs#L59

Can you please elaborate on your suggestion? I agree the format string approach isn't good, but I don't know of a better solution.